.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I think this should not be included, but would have to double check with @tatumalenko

Reply:
Yah I wouldn't either lol